modules/rostests/apitests/ntdll/RtlGetUnloadEventTrace.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for RtlGetUnloadEventTrace
+ * COPYRIGHT:   Copyright 2020 Mark Jansen (mark.jansen@reactos.org)
+ */
+
+#include "precomp.h"
+
+PRTL_UNLOAD_EVENT_TRACE
+NTAPI
+RtlGetUnloadEventTrace(VOID);
+
+#ifndef _WIN64
+C_ASSERT(sizeof(RTL_UNLOAD_EVENT_TRACE) == 84);
+C_ASSERT(sizeof(RTL_UNLOAD_EVENT_TRACE) * RTL_UNLOAD_EVENT_TRACE_NUMBER == 0x540);
+#endif
+
+static void Test_Dump()
+{
+    PRTL_UNLOAD_EVENT_TRACE TraceHead, Trace;
+    UINT n;
+
+    TraceHead = RtlGetUnloadEventTrace();
+    for (n = 0; n < RTL_UNLOAD_EVENT_TRACE_NUMBER; ++n)
+    {
+        ULONG ExpectSequence = n ? n : RTL_UNLOAD_EVENT_TRACE_NUMBER;
+
+        Trace = TraceHead + n;
+
+        ok(Trace->BaseAddress != NULL, "Got no BaseAddress for %u\n", n);
+        ok(Trace->SizeOfImage != 0, "Got no SizeOfImage for %u\n", n);
+        ok(Trace->Sequence == ExpectSequence,
+           "Wrong Sequence: %lu instead of %lu for %u\n", Trace->Sequence, ExpectSequence, n);
+        ok(Trace->TimeDateStamp != 0, "Got no TimeDateStamp for %u\n", n);
+        ok(Trace->CheckSum != 0, "Got no CheckSum for %u\n", n);
+        ok(!wcscmp(Trace->ImageName, L"GetUName.dLl"), "Wrong ImageName for %u: %S\n", n, Trace->ImageName);
+    }
+}
+
+#define TESTDLL "GetUName.dLl"
```

Comment:
Should the 2nd `L` in `GetUName.dLl` be uppercase? :thinking:

Reply:
the 3rd should be in lowercase you mean.

Reply:
No this is intentionally bad casing.

Reply:
Add a comment, like `// Intentional bad casing of 'getuname.dll'.`?